### PR TITLE
feat(schema): Rename schema fields

### PR DESCRIPTION
**Pull request by kkyeboah**
## Goal

Rename fields per team discussion

### CODE REVIEW

File: app/models/slate.py [hunk 1]
```diff
@@ -15,9 +15,9 @@ class SlateModel(BaseModel):
     Models a slate
     """
     id: str
-    requestID: str = None
-    experimentID: str = None
-    pageviewGUID: str = None
+    requestId: str = None
+    experimentId: str = None
+    pageviewGuid: str = None
```
Comment (bassrock) [inline app/models/slate.py:20]: Pretty sure we can remove this. Its not in our schema anymore.

File: schema.graphql [hunk 2]
```diff
@@ -182,13 +182,13 @@ type Slate {
     A guid that is unique to every API request that returned slates, such as `getSlateLineup` or `getSlate`.
     The API will provide a new request id every time apps hit the API.
     """
-    requestID: String!
+    requestId: String!
```
Comment (bassrock) [inline schema.graphql:185]: should these be ids while we are in here?

**Review by bassrock — DISMISSED** [review]
👍

**Comment by bassrock** [comment]
Tests are failing fyi @kkyeboah

**Review by bassrock — DISMISSED** [review]

**Review by bassrock — APPROVED** [review]

**Review by bassrock — APPROVED** [review]

**Review by bassrock — APPROVED** [review]

**Review by bassrock — APPROVED** [review]

**Review by mmiermans — APPROVED** [review]

**Pull request merged**